uber/automated_emails.py

```diff
@@ -1,28 +1,30 @@
-### WARNING - changing the email subject line for an email causes ALL of those emails to be re-sent!
-#   Note that since the EVENT_NAME is used in most of these emails, changing the event name mid-year
-#   could cause literally thousands of emails to be re-sent!
+# WARNING - changing the email subject line for an email causes ALL of those emails to be re-sent!
+# Note that since c.EVENT_NAME is used in most of these emails, changing the event name mid-year
+# could cause literally thousands of emails to be re-sent!
 
 from uber.common import *
 
+
 class AutomatedEmail:
     instances = OrderedDict()
 
```

Comment:
Just to clarify, you're accomplishing that by removing as many variable as possible from the __init__ script..?